src/integrationTest/sam.test.ts

```diff
@@ -254,6 +258,17 @@ describe('SAM Integration Tests', async () => {
                     assert(err.message.includes('directory already exists'))
                 }).timeout(TIMEOUT)
 
+                it('produces an initial launch configuration', async () => {
```

Comment:
It isn't appropriate to add mocks/fakes into the integration test space. You might be interested in setting up a test within the unit-ish tests under `/test` though.

This test suite already calls createSamApplication in the `before` block, so all that should be required is to check that a config already exists that matches expectations for this scenario.

Reply:
sure, moved